src/pages/ContributionsPage/ContributionsPage.tsx

```diff
@@ -131,8 +150,30 @@ const ContributionsPage = () => {
       },
     }
   );
+  const [updatedTeamSelText, setUpdatedTeamSelText] = useState<
+    string | undefined
+  >(data?.circles_by_pk?.team_sel_text);
```

Comment:
```suggestion
  >();
```

Reply:
You may need to reformat this file after committing these suggestions; try running `yarn prettier:fix`

Reply:
done